.github/workflows/test-platforms.yaml

```diff
@@ -0,0 +1,123 @@
+name: Test on platforms
+
+on:
+    push:
+        branches:
+            - 3.x
+            - master
+    pull_request:
+
+jobs:
+    test-mysql:
+        name: PHP ${{ matrix.php-version }} + MySQL + ${{ matrix.dependencies }} + ${{ matrix.variant }}
+
+        runs-on: ubuntu-latest
+
+        continue-on-error: ${{ matrix.allowed-to-fail }}
+
+        env:
+            DATABASE_URL: "mysql://root@127.0.0.1:3306/sonata_tests"
+
+        strategy:
+            matrix:
+                php-version:
+                    - '8.0'
+                dependencies: [highest]
+                allowed-to-fail: [false]
+                variant: [normal]
```

Comment:
Since you're not using variant, you could remove the key from the matrix

Reply:
I left them on purpose so if we change something on `test.yaml`, it would be easier to adapt, I can inline them.